validator-cli/src/ArbToEth/watcherArbToGnosis.ts

```diff
@@ -243,232 +306,90 @@ const watch = async () => {
-
-                // there's practically very little MEV on gnosis
-                // priority fee should just be a small amount to get the txn included in a block
-
-                let maxPriorityFeePerGas = BigNumber.from("3000000000"); // 3 gwei
-
-                // if claim is in min challenge period, we can use a higher priority fee
-                // this ensures the txn is always competitive during the censorship test (min challenge period)
-                if (claim.timestampClaimed < timeLocal - sequencerDelayLimit - epochPeriod) {
-                  try {
-                    const blockPendingGnosis = (await retryOperation(
-                      () => providerGnosis.getBlockWithTransactions("pending"),
-                      1000,
-                      10
-                    )) as BlockWithTransactions;
-                    // can't access actual gas used from pending block, consider all txns equal weight
-                    let maxPriorityFeePerGasAvg = BigNumber.from("0");
-                    for (const txn of blockPendingGnosis.transactions) {
-                      maxPriorityFeePerGasAvg = maxPriorityFeePerGasAvg.add(txn.maxPriorityFeePerGas);
-                    }
-                    maxPriorityFeePerGasAvg = maxPriorityFeePerGasAvg.div(blockPendingGnosis.transactions.length);
-                    if (maxPriorityFeePerGas.lt(maxPriorityFeePerGasAvg)) {
-                      maxPriorityFeePerGas = maxPriorityFeePerGasAvg;
-                    }
-                  } catch (e) {}
-
-                  // there's almost no MEV on gnosis
-                  // will update this default value if there is more MEV on gnosis in the future
-                  if (maxPriorityFeePerGas.lt(BigNumber.from("100000000000"))) {
-                    maxPriorityFeePerGas = BigNumber.from("100000000000"); // 100 gwei
-                  }
-
-                  if (maxPriorityFeePerGas.gt(maxFeePerGasProfitable)) {
-                    maxPriorityFeePerGas = maxFeePerGasProfitable;
-                  }
-                }
-
-                if (!inactive) {
-                  const txnChallenge = (await retryOperation(
-                    () =>
-                      veaOutbox.challenge(veaEpochOutboxCheck, claim, {
-                        maxFeePerGas: maxFeePerGasProfitable,
-                        maxPriorityFeePerGas: maxPriorityFeePerGas,
-                      }),
-                    1000,
-                    10
-                  )) as ContractTransaction;
-
-                  challengeTxnHashes.set(index, txnChallenge.hash);
-                  console.log(
-                    "challenging claim for epoch " + veaEpochOutboxCheck + " with txn hash " + txnChallenge.hash
-                  );
-                }
+                )) as ContractTransaction;
+                console.log("Epoch " + veaEpochOutboxCheck + " sendSnapshot called with txn " + txnSendSnapshot.hash);
+              }
```

Comment:
I think `challengeProgress.snapshot.status` should be updated to "pending" with the sendSnapshot txHash?

Reply:
Looks fixed 👍